locale/fr_FR/content/running_bookwyrm/install-prod-dockerless.md

```diff
@@ -65,12 +65,13 @@ GRANT ALL PRIVILEGES ON DATABASE bookwyrm TO bookwyrm;
 \q
 ```
 
-- Initialisez la base de données en exécutant `venv/bin/python3 manage.py migrate`
+- Migrer le schema de la base de données en exécutant `venv/bin/python3 manage.py migrate`
```

Comment:
The translation will have to happen in https://translate.joinbookwyrm.com/ -- these locale files are overwritten by the copy that's loaded from there